components/cf-typography/index.md

```diff
@@ -4,260 +4,232 @@ title:  "cf-typography"
-    </a>.
-    Oh, you might also want to visit this
-    <a class="icon-link icon-link__external-link" href="#">
-        <span class="icon-link_text">other organization's website</span>
-    </a> for further details.
-</p>
-<a class="icon-link icon-link__right" href="#">
-    Visit another section of our site
-</a>
+For more information, email
+<a class="a-link a-link__icon a-link__mail" href="#">
+    <span class="a-link_text">john.smith@cfpb.gov</span>
+</a>.
+Alternatively, you can
+<a class="a-link a-link__icon a-link__download" href="#">
+    <span class="a-link_text">download the info sheet</span>
+</a>.
+Oh, you might also want to visit this
+<a class="a-link a-link__icon a-link__external-link" href="#">
+    <span class="a-link_text">other organization's website</span>
+</a> for further details.
 
 #### Links with icons on the left (modifier)
 
-<a class="icon-link icon-link__mail icon-link__before" href="#">
-    <span class="icon-link_text">john.smith@cfpb.gov</span>
+<a class="a-link a-link__icon a-link__mail a-link__before" href="#">
+    <span class="a-link_text">john.smith@cfpb.gov</span>
 </a><br>
-<a class="icon-link icon-link__phone icon-link__before" href="#">
-    (123) 456-7890
+<a class="a-link a-link__icon a-link__phone a-link__before" href="#">
+    <span class="a-link_text">(123) 456-7890</span>
 </a>
 
-
 #### Non-wrapping icon links
 
-<p>
-    For more information, email
-    <a class="icon-link icon-link__mail icon-link__no-wrap" href="#">
-        <span class="icon-link_text">john.smith@cfpb.gov</span>
-    </a>.
-</p>
-
-
-### Styled link
-
-<a class="styled-link" href="#">
-    Default styled link
-</a>
-
+For more information, email
+<a class="a-link a-link__icon a-link__mail a-link__no-wrap" href="#">
+    <span class="a-link_text">john.smith@cfpb.gov</span>
+</a>.
 
 ### Jump link
 
-<a class="jump-link" href="#">
-    <span class="jump-link_text">Default jump link</span>
+<a class="a-link
+          a-link__jump
+          a-link__right" href="#">
+    <span class="a-link_text">Default jump link</span>
 </a>
 
 
-#### .jump-link__large (modifier)
+#### Jump link with large link modifier
 
-<a class="jump-link jump-link__large" href="#">
-    <span class="jump-link_text">Large jump link</span>
+<a class="a-link
+          a-link__jump
+          a-link__large
+          a-link__right" href="#">
+    <span class="a-link_text">Large jump link</span>
 </a>
 
 
-#### .jump-link__<icon> (modifier)
+#### Jump link with before link modifier
 
-<a class="jump-link jump-link__right" href="#">
-    <span class="jump-link_text">Jump link using the right icon from cf-icons</span>
+<a class="a-link
+          a-link__jump
+          a-link__before
+          a-link__left" href="#">
+    <span class="a-link_text">Jump link with icon on left</span>
 </a>
 
 
-#### .jump-link__before (modifier)
+#### Jump link with bg link modifier (only visible on small screens)
```

Comment:
Remove the second "link" on this line. Should just be "Jump link with bg modifier".